.editorconfig

```diff
@@ -1,15 +1,36 @@
 # EditorConfig is awesome: http://EditorConfig.org
 
 [*]
-indent_style = space
-indent_size = 2
-end_of_line = lf
-charset = utf-8
+indent_style = tab
+indent_size = 4
+
+shell_variant      = bash
```

Comment:
honestly I am not sure what to do about this, this is from #1434 and it is duplicated in this file twice... I might change this

Reply:
Actually it seems like the pre commit hook is outright ignoring our `editorconfig` file... I will look into this

Reply:
Removed uneeded dup, and fixed the hook ignoring the `editorconfig` (caused by older version of the hook)